### ISSUE TITLE
transit peptide annotation is missing residues

### ISSUE THREAD

**Issue by ValWood** [pombase/website#2281]
Here:

The protein feature viewer has a transit peptide imported from Uniprot
but the uniprot annotation doesn't specify the residues?

<img width="1072" alt="ticket missing residues" src="https://github.com/user-attachments/assets/e3031846-b8e5-4526-b342-ebdbc4718813">

**Comment by kimrutherford** [comment]
Which gene is that?

**Comment by kimrutherford** [comment]
> Which gene is that?

Found it: SPAC23C11.17

**Comment by ValWood** [comment]
Yep, I was just looking!

**Comment by kimrutherford** [comment]
The signal and transit peptide were some of the first features we loaded from the UniProt data into Chado.  I Added them to `pombe-embl/supporting_files/manual_so_term_annotations.tsv` but forgot about the residue positions:
https://github.com/pombase/website/issues/2115#issuecomment-2262250477

For the modifications I implemented a better system that reads directly from the UniProt data file into Chado without having to add annotation to `manual_so_term_annotations.tsv`.  That's a better plan because when we update the data file we don't have to manually edit a file in SVN.

I forgot to go back and do the same thing for the signal and transit peptides.  I'll implement that now.

(Note to self: only remove the annotations assigned by UniProt from the file.  There are some that are assigned by PomBase)

**Comment by kimrutherford** [comment]
> I forgot to go back and do the same thing for the signal and transit peptides. I'll implement that now.

I've made the code changes and I've removed the SO annotations with assigned by UniProt from the `manual_so_term_annotations.tsv` file in SVN.  But I was double checking things afterwards and found that we had a lot fewer signal peptide and transit peptide annotations now.

Some (20 or so) are gone because we filter the ones from the UniProt data file where the sequence is different to PomBase.  But it turns out that a while ago I accidentally added "assigned by UniProt" to a bunch of annotations that were made by you and used to be in the contig files.  So when I removed what I though were UniProt annotations from `manual_so_term_annotations.tsv` I removed a lot of yours too. 

I'm now trying a untangle things and restore the missing annotations from the old SVN files.
